workers/reporter/src/main/kotlin/reporter/ReporterWorker.kt

```diff
@@ -38,6 +39,8 @@ import org.jetbrains.exposed.sql.Database
 
 import org.slf4j.LoggerFactory
 
+private const val RUN_ID_LABEL: String = "runId"
```

Comment:
While I have no objections to add this label here for the reporter, just wondering whether there is already a concrete use case for this?
Alternatively, it would be an option to make the `NotifierOrtResultGenerator` class more generic, move it to `:workers:common` and use it for both notifier and reporter. This would reduce code duplication and provide a uniform set of labels for both workers.

Reply:
There is a feature request about having the _runId_ as label in the static web report. Therefore, it needs to be included also in the OrtResult used by the _reporter_ worker.
Refactoring: In order to get rid of the code duplication and have a place where all **common labels** can be specified, I extracted the duplicated code into a util class, which also adds the **common labels** that are used both by the _reporter_ and the _notifier_ worker.